### PR TITLE
Configurate calendar with FullCalendar

### PR DESCRIPTION
**Pull request by momja**
Adds a calendar to the front page of the website with the meetup data imported from an .ics file. Because of CORS issues, this includes a github action to fetch the calendar daily at 3AM PST and commit it to the repository. This is kind of "faking" a backend service for us.

### CODE REVIEW

File: typings.d.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+declare module '*.ics' {
```
Comment (momja) [inline typings.d.ts:1]: if we don't add typings, the build complains that they cannot find type:

```
Type error: Cannot find module '@/public/calendars/meetup.ics' or its corresponding type declarations.
```

**Comment by momja** [comment]
This is what it looks like:

Screenshot 2024-03-17 at 1 05 35 PM
Screenshot 2024-03-17 at 1 05 31 PM

**Comment by jof** [comment]
A very nifty hack! I like it.

**Comment by momja** [comment]
It was looking pretty bad on mobile, so I updated it to display as a list view instead:

image

**Comment by momja** [comment]
> It was looking pretty bad on mobile, so I updated it to display as a list view instead:
> 
> image

This can be improved further, but at a later date.

**Review by BrandonDHaskell — APPROVED** [review]
Looks great! Ship it!

File: public/calendars/bookwhen.ics
Comment (BrandonDHaskell) [inline public/calendars/bookwhen.ics:1]: Do wee need this file for the PR? Or is this updated daily @ 2AM?

Comment (BrandonDHaskell) [reply public/calendars/bookwhen.ics:1]: Asked in person. this is the file that will be updated @ 2AM.

**Review by BrandonDHaskell — APPROVED** [review]
Looks good. Ship it!

**Pull request merged**